Python/udemy_scrape.py

```diff
@@ -0,0 +1,60 @@
+from bs4 import BeautifulSoup  #Importing the Beautiful Soup Library
+import requests                #Importing the requests library
+import time                    #Importing the time library
+import csv                     #Importing the csv module
+import json                    #Importing the JSON library
+
+host = 'https://www.udemy.com'
+courses_api_endpoint = '/api-2.0/discovery-units/all_courses/?p=1&page_size=16&category_id=%d&source_page=category_page&locale=en_US&currency=eur&navigation_locale=en_US&skip_price=true&sos=pc&fl=cat'
+prices_api_endpoint = '/api-2.0/pricing/?course_ids=%s&fields[pricing_result]=price'
+headers = {'User-Agent':'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; WOW64; Trident/6.0)'}
+
+response = requests.get(host, headers=headers)
+front_page_soup = BeautifulSoup(response.text, 'lxml')
+top_categories = front_page_soup.select('a[data-purpose="category-card-title-link"]')
+
+with open('udemy.csv', 'w') as file:
```

Comment:
A person experienced in python would know what this function does, but a beginner won't. Please take this in account.